src/Core/Localization/CLDR/Reader.php

```diff
@@ -272,12 +272,19 @@ protected function mainPath($filename = '')
      * @param string $localeTag The wanted locale. Can be either a language code (e.g.: fr) of an IETF tag (e.g.: en-US)
      *
      * @return LocaleData
-     *
-     * @throws LocalizationException
      */
     protected function getLocaleData($localeTag)
     {
-        $xmlData = $this->getMainXmlData($localeTag);
+        try {
```

Comment:
wowowow big problem here, we said that we moved the try catch in `readLocaleData` so that the code is more understandable
you did add the try catch in `readLocaleData` but without removing this so it won't work as expected, please remove the try catch in `getLocaleData`